.travis.yml

```diff
@@ -10,7 +10,7 @@ cache:
 language: java
 
 jdk:
-  - oraclejdk9
+  - oraclejdk11
```

Comment:
Just wondering, should we make this openjdk11? I guess it might be causing the confusion about commercial features. Targeting the baseline seems reasonable

Reply:
Yes. Definitely.

Reply:
ah yeah that was unintentional.. I just needed something 11 :P can try to change to openjdk11 presuming travis has it